### ISSUE TITLE
Update tabulate or relax constraint

### ISSUE THREAD

**Issue by jmarrec** [conan-io/conan-package-tools#461]
### Description of Problem, Request, or Question

I am unsure if conan-package-tools needs **exactly** tabulate == 0.8.2, but perhaps trying a `>= 0.8.2` would be nice as I'm getting conflicts with `pandas` in particular.

https://github.com/conan-io/conan-package-tools/blob/0d2aab6a9bdfb2c6dca37d58171c9af6ba9e2c6f/cpt/requirements.txt#L3

### Environment Details

* Conan Package Tools Version: **0.31.0**
* Operating System: **Linux**
* Operation System Version: **Ubuntu 18.04**
* Conan version: **conan 1.22.2**
* Python version: **python 3.8.0**

### Steps to reproduce (Include if Applicable)

I have pandas 1.0.1. If you use the `pd.DataFrame.to_markdown()` you need to use tabulate.
conan-package-tools 0.31.1 forces a strict `tabulate == 0.8.2`, while tabulate is currently at `0.8.6`
As a result, I have incompatible packages and will get an error in pandas if I use 0.8.2

```
ImportError: Pandas requires version '0.8.3' or newer of 'tabulate' (version '0.8.2' currently installed).
```

And vice versa:

```
$ pip install --upgrade tabulate
Processing /home/julien/.cache/pip/wheels/86/8c/d9/3b6c2f55efaefe656c52b725d2fdc5669085c83444979db8b3/tabulate-0.8.6-py3-none-any.whl
ERROR: conan-package-tools 0.31.0 has requirement tabulate==0.8.2, but you'll have tabulate 0.8.6 which is incompatible.
```

**Comment by jmarrec** [comment]
Thanks @uilianries. I noticed that `six` also has the same status:
```
conan-package-tools 0.31.1 has requirement six<1.13.0,>=1.10.0, but you have six 1.14.0.
```

**Comment by uilianries** [comment]
Thanks for reporting it. I gonna check Conan first, because they are using `six` there too.

**Comment by uilianries** [comment]
Implemented by #464 and #463
